src/js/utils/url.js

```diff
@@ -85,6 +85,10 @@ export const parseUrl = function(url) {
     document.body.removeChild(div);
   }
 
+  if (details.protocol === '') {
```

Comment:
seems like this belongs in separate PR.

Reply:
You are right. This is a separate topic.
It's my first time committing to a bigger project with such a refined build tool chain and I did not want to (and I believe it didn't let me) push failing code. Deemed that more important than singling out a secondary fix which is closed due to inactivity anyway and wait for it to be accepted before committing this PR.

If you prefer I can revert this here and add it to a separate PR.

Reply:
Thanks for contributing!
Making multiple PRs is generally better because then a single topic isn't blocking another topic that has been approved and the discussion can focus on the thing issue at hand.

Reply:
Ok. I will move it out from here.